### PR TITLE
Text to Speech using Web Speech API in JavaScript

### PR DESCRIPTION
**Pull request by zolomohan**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by zolomohan** [comment]
This fixes #1127

File: articles/text-to-speech-in-javascript/index.md [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+document.querySelector("#talk").addEventListener("click", () => {
+  speech.text = document.querySelector("textarea").value;
+});
+```
+
+**Volume**:
+
+The `volume` property gets and sets the volume of the utterance. It is a float that represents the volume value, between 0 (lowest) and 1 (highest). The default value is 1 if this property is unset.
+
+Let's add a `onInput` listener to the `volume` range slider and adjust the `volume` property when the value of the slider changes. We've already set the min, max and default value of the slider in the HTML tag.
```
Comment (cyberShaw) [inline articles/text-to-speech-in-javascript/index.md:143]: > a `onInput` listener

Wouldn't "an `onInput` listener" be more appropriate?

File: articles/text-to-speech-in-javascript/index.md [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+      <button id="pause" class="btn btn-warning mt-5 me-3">Pause</button>
+      <button id="resume" class="btn btn-info mt-5 me-3">Resume</button>
+      <button id="cancel" class="btn btn-danger mt-5 me-3">Cancel</button>
+    </div>
+  </body>
+  <script src="./textToSpeech.js"></script>
+</html>
+```
+
+![HTML Web Page](start_screen.png)
```
Comment (cyberShaw) [inline articles/text-to-speech-in-javascript/index.md:101]: According to convention, the image path should be, 

```md
![HTML Web Page](/engineering-education/text-to-speech-using-web-speech-api-in-javascript/start_screen.png)
```

**Review by cyberShaw — CHANGES_REQUESTED** [review]
Really amazing article, @zolomohan! Keep up the good work! I have a couple things to make sure before we pass this article off to final review. Please check them out!

File: articles/text-to-speech-in-javascript/index.md (hunk 1 above)
Comment (zolomohan) [reply articles/text-to-speech-in-javascript/index.md]: yep. not sure how I missed it 😅

**Comment by zolomohan** [comment]
@cyberShaw Made those changes.

**Comment by zolomohan** [comment]
@cyberShaw Thank you. I made those changes.

**Comment by cyberShaw** [comment]
> @cyberShaw Thank you. I made those changes.

Thank you for making the changes, @zolomohan! This article is now ready for final review.

**Comment by louisefindlay23** [comment]
@hectorkambow, this article is ready for publication.

**Review by louisefindlay23 — APPROVED** [review]
@hectorkambow, this article is ready for publication.

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**